### PR TITLE
Updates for latest metal-devtools changes

### PR DESCRIPTION
**Pull request by bryceosterhaus**
Changing the API slightly for metal-devtools to work better with components being added while the page has already been rendered. Once merged, I will release a new version of devtools.

### CODE REVIEW

**Review by jbalsas — APPROVED** [review]
Hey @bryceosterhaus, https://github.com/bryceosterhaus/metal-devtools is looking awesome! Do you think it would be time to move it to the metal organization to officially endorse it?

I don't have merge rights, so I just left a simple comment although this looks good to me :)

File: packages/metal-component/src/Component.js [hunk 1]
```diff
@@ -439,8 +439,8 @@ class Component extends EventEmitter {
 		}
 		const instance = new Ctor(config, false);
 
-		if (window.__METAL_DEV_TOOLS_HOOK__) {
-			window.__METAL_DEV_TOOLS_HOOK__(instance);
+		if (window.__METAL_DEV_TOOLS_HOOK__ && window.__METAL_DEV_TOOLS_HOOK__.addRoot) {
+			window.__METAL_DEV_TOOLS_HOOK__.addRoot(instance);
```
[the remark this answers is not in the document]
Comment (bryceosterhaus) [reply packages/metal-component/src/Component.js]: Yeah you are totally right about having the backwards compatibility. Ill update this PR.

~~As far as moving devtools under the metal repo, I think it should wait until it is adopted a bit more so we can get to a more stable release of it. Im sure there are still plenty of issues I haven't encountered yet.~~

**Comment by bryceosterhaus** [comment]
@jbalsas  For moving devtools under the metal repo, I think it should wait until it is adopted a bit more so we can get to a more stable release of it. Im sure there are still plenty of issues I haven't encountered yet.

File: packages/metal-component/src/Component.js [hunk 2]
```diff
@@ -440,7 +440,9 @@ class Component extends EventEmitter {
 		const instance = new Ctor(config, false);
 
 		if (window.__METAL_DEV_TOOLS_HOOK__) {
-			window.__METAL_DEV_TOOLS_HOOK__(instance);
+			let hook = window.__METAL_DEV_TOOLS_HOOK__.addRoot ? window.__METAL_DEV_TOOLS_HOOK__.addRoot : window.__METAL_DEV_TOOLS_HOOK__;
```
Comment (pragmaticivan) [inline packages/metal-component/src/Component.js]: Since you're not generating a new binding on hook anywhere, that may be a const.

Comment (bryceosterhaus) [reply packages/metal-component/src/Component.js]: Right you are, just updated commit

**Comment by jbalsas** [comment]
Sounds good, @bryceosterhaus, I'll make sure to at least start passing the word around to get more people using it so we can evolve it faster.

Will merge this as soon as possible! :)

**Comment by bryceosterhaus** [comment]
~~Closing for now. Will re-open if need. I am investigating a different means of accomplishing this.~~

**Pull request closed without merging**